src/components/AddressSearch/index.js

```diff
@@ -342,6 +364,26 @@ function AddressSearch(props) {
         };
     }, []);
 
+    const listEmptyComponent = useCallback(
```

Comment:
I think we can use `useMemo` here since we just want to store the value

Reply:
@luacmartins I've changed function above to memo, but I think here I think it's safer to use callback.